core/src/main/java/com/datastrato/gravitino/Configs.java

```diff
@@ -19,9 +19,19 @@ public interface Configs {
   String DEFAULT_ENTITY_KV_STORE = "RocksDBKvBackend";
   String ENTITY_KV_STORE_KEY = "gravitino.entity.store.kv";
 
-  String DEFAULT_ENTITY_RELATIONAL_STORE = "MySQLBackend";
+  String DEFAULT_ENTITY_RELATIONAL_STORE = "JDBCBackend";
   String ENTITY_RELATIONAL_STORE_KEY = "gravitino.entity.store.relational";
 
+  String ENTITY_RELATIONAL_JDBC_BACKEND_DB_TYPE_KEY =
+      "gravitino.entity.store.relational.jdbc.dbType";
+  String DEFAULT_ENTITY_RELATIONAL_JDBC_BACKEND_DB_TYPE = "mysql";
```

Comment:
Do we need dbType?

Reply:
In my opinion, add this parameter makes sense. Because different databases may have differences in the SQL syntax and parameters, we can add this parameter to facilitate switching related implementations.

Reply:
Is the parameter enough? I just feel that this may not elegant. You can refer to some other systems.

Reply:
`NetEase` has a similar data lake metadata system named `Amoro`. In their implementation, the db type parameter is also added for the JDBC backend for differentiation. See: https://github.com/NetEase/amoro/blob/926185ed102ce30f4fbc7ffd29d4dad3ea6f2e1e/ams/server/src/main/java/com/netease/arctic/server/ArcticManagementConf.java#L229. The problem with using `Driver` is that we may need to maintain a mapping list. For example, `MySQL` supports both `com.mysql.cj.jdbc.Driver` and `com.mysql.jdbc.Driver`, which is even more redundant than the current method.

Reply:
OK, `DBType` vs `DatabaseType`. What do you prefer?

Reply:
I have no particular inclination. If we use the `jdbcDBType`, just clearly indicate the meaning of `DB` in the description.

Reply:
> `NetEase` has a similar data lake metadata system named `Amoro`. In their implementation, the db type parameter is also added for the JDBC backend for differentiation. See: https://github.com/NetEase/amoro/blob/926185ed102ce30f4fbc7ffd29d4dad3ea6f2e1e/ams/server/src/main/java/com/netease/arctic/server/ArcticManagementConf.java#L229. The problem with using `Driver` to distinguish database is that we may need to maintain a mapping list. For example, `MySQL` supports both `com.mysql.cj.jdbc.Driver` and `com.mysql.jdbc.Driver`, which is even more redundant than the current method.

@xloya This does not explain why we need `jdbcDBType`. If we introduce `jdbcDBType`, we may need more check to keep `jdbcType` consitent with `jdbcDriver`. Because user may set `jdbcType=Oracle` and set `jdbcDriver=com.mysql.jdbc.Driver` I think a `jdbcDriver` is enough, we can infer the `jdbcDBType` by the `jdbcDriver.`

Reply:
@coolderli To be honest, I don't actually want to maintain a mapping list of `driver names` and `databases` in the code. Will it be a problem for users to use the wrong driver? Because users cannot use it directly without knowing it. They need to put the driver in the `lib` directory. But I think the subsequent demand for `jdbcDBType` can be clarified. Not all databases have the same SQL syntax, so we need to customize SQLs for the special syntax of different databases.

Reply:
@qqqttt123 The SQL syntax used by Iceberg is relatively simple, so there is no need to pay attention to the database type in implementation. If there is currently a big controversy about this parameter, we can not add this parameter now. If we need to know the database type, we can truncate it from the jdbc url (such as jdbc:mysql://xxx).